srcts/src/bindings/outputAdapter.ts

```diff
@@ -22,7 +22,9 @@ class OutputBindingAdapter {
     // onResize with a version that does a makeResizeFilter on the element.
     if (binding.resize) {
       this.onResize = makeResizeFilter(el, function (width, height) {
-        binding.resize(el, width, height);
+        if (typeof binding.resize === "function") {
+          binding.resize(el, width, height);
+        }
```

Comment:
`binding.resize` is not guaranteed to be defined. Safe-guard the method call

Reply:
Maybe try `binding.resize!(`